build_msvc/bitcoin_config.h

```diff
@@ -53,6 +53,9 @@
 /* define if the Boost::Process library is available */
 #define HAVE_BOOST_PROCESS /**/
 
+/* define if external signer support is enabled (requires Boost::Process) */
+#define ENABLE_EXTERNAL_SIGNER 1 /**/
```

Comment:
Just defining or setting it to 1 always confused me. As does MSVC. But it sounds reasonable... @sipsorcery?

Reply:
I think it's supposed to be either the default or a fixed setting indicating what msvc should be building. Either way, having `HAVE_BOOST_PROCESS` not defined above and still defining `ENABLE_EXTERNAL_SIGNER` is a conflict because we can't have external signer support without boost process.

Reply:
I think msvc is the same as the other compilers. If you want to set a pre-processor define to act like a boolean flag there's no need to set a value on it. `#define ENABLE_EXTERNAL_SIGNER` is all you need if you're only checking for its existence with `#ifdef ENABLE_EXTERNAL_SIGNER`.

Reply:
Ok, I'll use that.